templates/job-submit.php

```diff
@@ -16,6 +16,7 @@
 }
 
 global $job_manager;
+$captcha_version = get_option( 'job_manager_recaptcha_version' );
```

Comment:
Might be worth adding a `get_captcha_version()` function somewhere that puts getting the option or returning the default value in one place.

Reply:
Done in 260ba28f0540e2a73d48dda4b4bae2e99e3e56bf